### DISCUSSION TITLE
installation on windows

### DISCUSSION THREAD

**Discussion by RedemptionC** [pnpm/pnpm#4732] [Q&A]
Hi there,I'm trying to install pnpm on windows(10)
I installed it using npm `npm install pnpm -g`
it succeeded:
```
╰─ npm list -g --depth 0                                                                                          ─╯
C:\Users\namei\AppData\Roaming\npm
`-- pnpm@7.1.0
```
But I still can't use it:
```
╰─ pnpm                                                                                                           ─╯
pnpm: The term 'pnpm' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```
I even tried rebooting windows,still doesn't work out 
what should I do ?

**Comment by RedemptionC** [comment]
@iamstarkov 
@rstacruz 
@zkochan

**Reply by zkochan** [reply]
If you used npm to install it, it is an issue with npm.

You can use another installation method, there a few options:

https://pnpm.io/installation

**Reply by RedemptionC** [reply]
Hi @zkochan ,thanks for reply !
But
```
╰─ iwr https://get.pnpm.io/install.ps1 -useb | iex                                                                ─╯
Downloading '@pnpm/win-x64' from 'npmjs.com' registry...                                                             
Extracting downloaded '7.1.0' archive...

/usr/bin/tar: Cannot connect to C: resolve failed
Running setup...

Invoke-Expression: Cannot validate argument on parameter 'FilePath'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
```

**Reply by RedemptionC** [reply]
Oh , I found that I can do it in powershell lower than powershell7
BTW,it seems that we should add PNPM_HOME to PATH manually

**Reply by zkochan** [reply]
if you use that script, it should be added automatically. But probably you should open a new terminal

**Reply by RedemptionC** [reply]
Maybe...
But I think it's nessary to claim that that script is not for powershell 7.x
![image](https://user-images.githubusercontent.com/17563754/168426971-0fa6518e-466a-40c4-9e3a-034eab4f6fdc.png)

**Reply by zkochan** [reply]
If it doesn't work with powershell 7, then it should be fixed.
